bitsail-connectors/connector-hbase/src/main/java/com/bytedance/bitsail/connector/hbase/auth/KerberosAuthenticator.java

```diff
@@ -0,0 +1,110 @@
+package com.bytedance.bitsail.connector.hbase.auth;
```

Comment:
Missing Bytedance license? Apply for other new created files.

Reply:
"Licensed under the Apache License". I saw this statement is in all other files. What bytedance license did you mean?

Reply:
Yes "Licensed under the Apache License" in specific. BitSail PR even has an auto license check. I see the current commit (18th) fails on it because `KerberosAuthenticator` and others miss such license.